Woof/Woof/View/SitterFlow/SitterProfileView/SitterProfileView.swift

```diff
@@ -28,6 +28,9 @@ struct SitterProfileView: View {
                         Spacer()
 
                         Button(saveButtonLabelText) {
+                            Task {
+                                try await viewModel.upload()
+                            }
```

Comment:
Does it make sense to move the network calling and saving data locally in ViewModel and don't expose them to View? We can make these methods as private and add one general method where the all logic will be done.
WDYT?

Reply:
Fixed